cpg-analysis/src/test/kotlin/de/fraunhofer/aisec/cpg/analysis/SizeEvaluatorTest.kt

```diff
@@ -49,7 +51,10 @@ class SizeEvaluatorTest {
                 listOf(topLevel.resolve("size.java").toFile()),
                 topLevel,
                 true
-            )
+            ) {
+                it.registerLanguage(JavaLanguage())
+                    .registerPass(JavaExternalTypeHierarchyResolver())
```

Comment:
shouldn't this pass automatically be registered with the language?

Reply:
No, you have to register it. The annotation `RequiredFrontend` means that the pass won't be executed if the `JavaLanguageFrontend` isn't registered.

Reply:
Ah ok. I thought we added an annotation that supports automatic registering of a "default" pass once a language is registered. If not, we should probably have it :) 

cc @maximiliankaul

Reply:
A frontend can require passes (and passes can require other passes).
The `JavaLanguageFrontend` could be annotated with `RegisterExtraPass` which would make it always execute the `JavaExternalTypeHierarchyResolver`. Is this pass a pass which should always be executed (for Java)?

Reply:
> A frontend can require passes (and passes can require other passes).
> The `JavaLanguageFrontend` could be annotated with `RegisterExtraPass` which would make it always execute the `JavaExternalTypeHierarchyResolver`. Is this pass a pass which should always be executed (for Java)?

Yes, it is.